### PR TITLE
Added boilerplate automation

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Added boilerplate automation workflow. It will trigger from the boilerplate repo. 
- Updated boilerplate manually 
- Boilerplate migration from lyft to flyte

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by pmahindrakar-oss** [comment]
As per our discussion.
The boiler plate files which are not getting used in the flytectl repo is there a way we can exclude those through the update command or in the repo by adding it to gitignore. eg : docker image build files which are currently not being used in flytectl

**Review by pmahindrakar-oss — APPROVED** [review]
Was there an update to .gitignore required ?

**Comment by evalsocket** [comment]
@pmahindrakar-oss no
